Sources/AltSwiftUI/Source/ViewProperties/ViewPropertyBuilder.swift

```diff
@@ -410,11 +405,11 @@ extension View {
     ///   - y: A vertical offset you use to position the shadow relative to
     ///     this view.
     public func shadow(color: Color = Color(white: 0, opacity: 0.33), radius: CGFloat, x: CGFloat = 0, y: CGFloat = 0) -> Self {
+        // swiftlint:disable:previous function_default_parameter_at_end
```

Comment:
Interfaces are not totally following this rule (follow SwiftUI API, readability, etc.), may be better to disable

Reply:
Disabled